pkg/secrethub/client.go

```diff
@@ -42,6 +46,27 @@ type Client struct {
 	// repoindexKeys are the keys used to generate blind names in the repo.
 	// These are cached
 	repoIndexKeys map[api.RepoPath]*crypto.SymmetricKey
+
+	appInfo *AppInfo
+}
+
+// AppInfo contains information about the application that is using the SecretHub client.
+// It is used to identify the application to the SecretHub API.
```

Comment:
Do we need users of the library to set this themselves? If so, consider adding a comment like: 

`It is used to identify the application to the SecretHub API, helping us a lot when analyzing errors and usage. So please provide this information when integrating with this libary.`

Reply:
In the future, we could also (perhaps) expose an analytics route that allows integration builders to see how actively their 'App' is being used, e.g. call `api.secrethub.io/v1/analytics/apps/secrethub-cli` responds with e.g. the number of API calls made for that app in the last 30 days. 

Not a priority, at all, but something cool to consider